GitUI/CommandsDialogs/BrowseDialog/GitStatusMonitor.cs

```diff
@@ -198,8 +198,16 @@ private GitStatusMonitorState CurrentStatus
                     case GitStatusMonitorState.Running:
                         {
                             _timerRefresh.Start();
-                            _workTreeWatcher.EnableRaisingEvents = true;
-                            _gitDirWatcher.EnableRaisingEvents = !_gitDirWatcher.Path.StartsWith(_workTreeWatcher.Path);
+                            if (Directory.Exists(_workTreeWatcher.Path))
+                            {
+                                _workTreeWatcher.EnableRaisingEvents = true;
+                            }
+
+                            if (!Directory.Exists(_gitDirWatcher.Path))
```

Comment:
I have not digged into the details. Shall this really be done only if the directory does not exist yet?

Reply:
remains of my testing....